Sanitize utils.safetyObject output from big objects

akazakou opened the pull request:
## :memo: Sanitize utils.safetyObject output from big objects

In some cases, we should pass into service `schemaMods` parameters a big amount of data. To example: some instances of [SharedArrayBuffer](https://developer.mozilla.org/ru/docs/Web/JavaScript/Reference/Global_Objects/SharedArrayBuffer) class with data more than 100MB. In that case on service starting Moleculer.js tries to post that data into output/databus and it causes performance in `utils.safetyObject` to convert this amount of data into JSON.

### :dart: Relevant issues
- No related issues

### :gem: Type of change
- [x] New feature (non-breaking change which adds functionality)

### :scroll: Example code
```js
    const broker = new ServiceBroker({ logLevel: "debug" });
    broker.createService(Service, {
        name: 'maxmind',
        settings: {
            service: {
                database: new Buffer(Array.from({ length: 100 * 1024 * 1024 })),
            },
        },
    });
    broker.start();
``` 

## :vertical_traffic_light: How Has This Been Tested?

Example of code to reproduce that issue provided in "Example code" section

## :checkered_flag: Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] **I have added tests that prove my fix is effective or that my feature works**
- [x] **New and existing unit tests pass locally with my changes**
- [x] I have commented my code, particularly in hard-to-understand areas

icebob commented:
On the other hand, the default value should be null & handle as infinite otherwise it causes breaking change.

akazakou commented:
@icebob Changes from ed821e7 commit
* Moved `maxSafeObjectSize` to service broker options. The default value for broker options now is `null` - checks will be disabled
* Updated unit tests to support new broker options in exists tests. 
* Added changes in CHANGELOG.md
